presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1215,12 +1232,19 @@ private void executeIrreversibleMetastoreOperations()
                 }
             }
             if (!suppressedExceptions.isEmpty()) {
+                StringBuilder message = new StringBuilder();
+                if (deleteOnly && !anySucceeded) {
+                    message.append("The following metastore delete operations failed: ");
+                }
+                else {
+                    message.append("The transaction didn't commit cleanly. All operations other than the following delete operations were completed: ");
+                }
+                message.append(failedIrreversibleOperationDescriptions.stream()
```

Comment:
I'd put this on the same line since there is only one action. It is short if you static import:
```java
message.append(failedIrreversibleOperationDescriptions.stream().collect(joining("; ")));
```
Or use `Joiner`, which supports `StringBuilder`:
```
Joiner.on("; ").appendTo(message, failedIrreversibleOperationDescriptions);